asn1crypto/core.py

```diff
@@ -166,6 +166,15 @@ def load(encoded_data, strict=False):
     return Asn1Value.load(encoded_data, strict=strict)
 
 
+def _reconstruct_asn1value(asn1crypto_cls, der_bytes):
```

Comment:
Can we rename this to `unpickle_helper()`? Since it must be importable, I'd rather it not be private, and the name should make it clear what it is used for.

Reply:
Hi @wbond, I renamed the function to `unpickle_helper` as requested.